drivers/spi/CMakeLists.txt

```diff
@@ -41,6 +41,8 @@ zephyr_library_sources_ifdef(CONFIG_SPI_PW		spi_pw.c)
 zephyr_library_sources_ifdef(CONFIG_SPI_SMARTBOND   spi_smartbond.c)
 zephyr_library_sources_ifdef(CONFIG_SPI_OPENTITAN	spi_opentitan.c)
 zephyr_library_sources_ifdef(CONFIG_SPI_NUMAKER		spi_numaker.c)
+zephyr_library_sources_ifdef(CONFIG_SPI_AMBIQ		spi_ambiq.c)
```

Comment:
Rebase artifact?

Reply:
Yes.  Still trying to clean it up ...

Reply:
@yonsch - ... Got it!  Can you approve the review?